### DISCUSSION TITLE
Question: Customizing the row height?

### DISCUSSION THREAD

**Discussion by azer** [company-mode/company-mode#1188] [Q&A] [answered]
Is it possible to increase the size of a row in company-mode ? If no, it'd be great.

**Comment by dgutov** [comment]
You mean the popup? It's rendered using an overlay, so it's not at all possible, sorry.

**Comment by azer** [comment]
@dgutov I mean the rows in the popup, line height on the other words

**Comment by dgutov** [comment]
Right. Impossible, with the current rendering method.

**Comment by dgutov** [comment, marked as answer]
The built-in popup is rendered using an overlay, so it's not possible OOTB.

But these days we have child frame popup packages (in the GUI Emacs), so one can look into using https://github.com/tumashu/company-posframe/ or https://github.com/sebastiencs/company-box and changing the height by customizing some stuff there.

E.g. using `company-posframe-show-params` or the `company-box-candidate` face.
